src/Calendar/components/TimeGrid/TimeGrid.js

```diff
@@ -181,22 +180,24 @@ const TimeGrid = React.forwardRef(
             {renderTimes()}
           </div>
           <div className={makeClass('time-grid__grid')}>
-            <div
-              className={currentTimeIndicatorClass}
-              style={{
-                top: `${getTopOffset({
-                  stepMinutes,
-                  date: currentTime,
-                  stepHeight,
-                })}px`,
-              }}
-              ref={timeIndicatorRef}
-            >
-              <span className={`${currentTimeIndicatorClass}__time`}>
-                {moment().format('h:mma')}
-              </span>
-              <div className={`${currentTimeIndicatorClass}__line`} />
-            </div>
+            {!disableCurrentTimeIndicator && (
```

Comment:
My first instinct is to make this a render prop instead so `renderTimeIndicator` then they could make it render `null` if they wanted to disable it. Would also allow the user to change the time indicator to something they would prefer.

Reply:
wondering if the renderProp can be added later, if it seems that use-case is needed?

Reply:
I like the idea of allowing users to change the look and feel of the current time indicator. However, I think it's clearer to have props for disabling specific features instead of implementing it via side effects from render functions

Reply:
agree with @cservenka, leaning towards the more straightforward implementation